include/mbedtls/asn1write.h

```diff
@@ -152,6 +152,21 @@ int mbedtls_asn1_write_bool( unsigned char **p, unsigned char *start, int boolea
  */
 int mbedtls_asn1_write_int( unsigned char **p, unsigned char *start, int val );
 
+/**
+ * \brief           Write a given string tag and
```

Comment:
The documentation should be expanded here. Suggestion: `Write a string in ASN.1 format using a specific encoding tag.`